durablefunctionsmonitor-vscodeext/src/az-func-as-a-graph/traverseFunctionProjectUtils.ts

```diff
@@ -304,5 +320,5 @@ export class DotNetBindingsParser {
     static readonly signalRConnInfoParamsRegex = new RegExp(`"([^"]+)"`);
     static readonly eventGridParamsRegex = new RegExp(`"([^"]+)"(.|\r|\n)+?"([^"]+)"`);
 
-    static readonly isOutRegex = new RegExp(`\\]\\s*(out |ICollector|IAsyncCollector).*?(,|\\()`, 'g');
+    static readonly isOutRegex = new RegExp(`^\\s*\\]\\s*(out |ICollector|IAsyncCollector).*?(,|\\()`, 'g');
```

Comment:
What is `isOutRegex` used for and how does this change modify the behavior?

Reply:
It is used to determine whether a binding is an input or an output binding.
Sometimes it produced incorrect results (when e.g. a function has multiple bindings, and one is input and next one is output).
To fix that, it needs to be anchored to the beginning of the string.